### DISCUSSION TITLE
adding new model for moobloom

### DISCUSSION THREAD

**Discussion by aarongiesbrecht** [Faboslav/friends-and-foes#155] [Q&A]
Hi, I'm looking o create a new model for the moobloom using blockbench so that I can have the moobloom match the look of my cows/mooshrooms. currently I have a resource pack that affects the other cow type mods but as I can seem to get it to affect the mooshroom I figured I would just try to create my own model. would it be as simple as creating a model and adding the model file to the blockbench directory?
alternatively if there is something im missing for getting a resource pack to piont towards any of the mobs from this pack I would love any advice

**Comment by Faboslav** [comment]
Hello, as i didnt see this i am too late for the response, i am sorry. For the context moobloom uses a cow model, so if you are able to change that, then it should affect moobloom too :).
